extension-config.json

```diff
@@ -7,14 +7,417 @@
       "type": "static",
       "destination": "frontend",
       "params": {
-        "value": "DE,AT,FR,GB"
+        "value": "DE,US,AT,FR,GB"
       }
     },
     "addressFields": {
-      "type": "static",
-      "destination": "frontend",
+      "type": "admin",
+      "destination": "both",
+      "default": {
+        "locales": {
+          "default": {
+            "firstName.label": "First name",
+            "firstName.placeholder": "Please enter your first name",
+            "lastName.label": "Last name",
+            "lastName.placeholder": "Please enter your last name",
+            "street1.label": "Street and number",
+            "street1.placeholder": "",
+            "street2.label": "",
+            "zipCode.label": "",
+            "zipCode.placeholder": "",
+            "city.label": "City",
+            "city.placeholder": "",
+            "note.validationRules.max_length_10": "You entered to many characters!!!",
+            "note.validationRules.no_numbers": "You should not add any numbers!!!"
+          },
+          "de-DE": {
+            "firstName.label": "Vorname",
+            "firstName.placeholder": "Bitte trage deinen Vorname ein",
+            "lastName.label": "Nachname",
+            "lastName.placeholder": "Bitte trage deinen Nachnamen ein",
+            "street1.label": "Straße und Hausnummer",
+            "street1.placeholder": "",
+            "street2.label": "",
+            "zipCode.label": "",
+            "zipCode.placeholder": "",
+            "city.label": "Stadt"
+          }
+        },
+        "fields": {
+          "firstName": {
```

Comment:
After talking to our PO I came to the conclusion that we won't.
This would only make people think, that all html attributes can be used, but not all will be implemented. All attributes should be achievable by using form actions and later validation.
Adding them would not benefit us, because we would need to implement validation errors as well then.